### ISSUE TITLE
Not running per subject -- skipping sessions without anat files

### ISSUE THREAD

**Issue by keithcdodd** [brainlife/app-fmriprep#19]
Hello,

Let's say I have a subject with 4 sessions. They get anatomicals on sessions 1 and 2. Functional scans are done on all four sessions.

On Docker I can run fmriprep per participant and it handles all sessions and all tasks.

On fmriprep on brainlife through pipeline it appears I have to specify only one functional task at a time. Even when I do that and it runs, it does not appear to process any functional tasks in sessions that do not include an anat file. So functional tasks in sessions 2 and 4 are ignored completely.

Is there a way I can just have it run per subject and handle all sessions all tasks? Or at least be able to get the functional tasks in sessions 2 and 4 to work? Or is best case scenario to copy over the anat files into sessions 2 and 4?

Thanks!

**Comment by keithcdodd** [comment]
Hmm, sadly that didn't seem to do anything. It did not submit new tasks from that and instead acts as if all those processes were already done from before, or it is waiting on more data. I wonder if it doesn't know how to match them up properly? The number of files do not align. Some subjects did not get the functional scans.

What can I do?

Thanks!
